doc/ref_arch/openstack/chapters/chapter06.md

```diff
@@ -28,26 +28,30 @@ Chapter 2 gathers all requirements and recommendations regarding security topics
 ## 6.3 NFVI and VIM Security
 
 OpenStack security guide:
-https://docs.openstack.org/security-guide/introduction/introduction-to-openstack.html
+https://docs.openstack.org/security-guide/introduction/introduction-to-openstack.html. In the section "[Sceurity boundaries and threats](https://docs.openstack.org/security-guide/introduction/security-boundaries-and-threats.html)" there is extensive description on secuirty domains, threat classifications, and attack vectors. The following only touches on some of the topics and at a high level.
+
+The handling of these security incidents requires various levels of logging of key infrastructure, processes, and use behaviour. These logs have to ce continuously monitored and analysed with alerts created for anomalies. The resources for logging, monitoring and alerting also need to logged and monitored and corrective actions taken so that they are never short of the needed resources (sec.mon.015).
 
 <a name="6.3.1"></a>
 ### 6.3.1 Platform Access
 
-#### 6.3.1.1 Identity
+#### 6.3.1.1 [Identity Security](https://docs.openstack.org/security-guide/identity.html)
 The OpenStack Identity service (Keystone) provides identity, token, catalog, and policy services for use specifically by services in the OpenStack family. Identity service is organized as a group of internal services exposed on one or many endpoints. Many of these services are used in a combined fashion by the front end.
 
 OpenStack Keystone can work with an Identity service that your enterprise may already have, such as LDAP with Active Directory.  In those cases, the recommendation is to integrate Keystone with the cloud provider's Identity Services.  
```

Comment:
Do you think we should actually recommend where multiple individual named users will access OpenStack that a suitable corporate directory services system such as MS AD is used since the built-in OpenStack authentication services could create shadow IT and detachment from any joiners and leavers processes?

Reply:
@iangardner22 The last sentence does recommend integrating with the "cloud provider's Identity Services." Do you want to specifically recommend MS AD or somehow change to "strongly recommended" or "must integrate"?

Reply:
No that makes sense - sorry missed it